### ISSUE TITLE
Crashed persistent

### ISSUE THREAD

**Issue by monstrodopantano** [fougner/ulauncher-pidgin#1]
Hi,

I would like to report a crash in extension. The solution I can do is to remove and install again, it happens that if I close the session or restart my computer, the crash occurs again. I've removed and installed it several times, the same problem always occurs.

SO: Debian 10 - Gnome

![ulauncher](https://user-images.githubusercontent.com/7289939/90249203-6c894580-de10-11ea-986d-f15c9c9f00ae.png)

**Comment by fougner** [comment]
Is Pidgin running when ulauncher is starting? The previous implementation was very naive and crashed if it can't get the DBus interface when loading the extension. I added some error handling in the latest changes, please try it and see if that resolves your issue. If the issue persist, please provide a stacktrace by launching ulauncher with the verbose flag, `ulauncher -v` and reproduce the crash.

**Comment by bluemax75** [comment]
Hi, I had the same issue. Now it is working for me. Thanks!

**Comment by monstrodopantano** [comment]
Hi, 
Sorry the long time response, yes error persist and yes pidgin not start with boot linux only ulancher this occurred used mode verbose for starting the app without the pidgin started first.

`2020-08-25 09:14:24,727 | DEBUG | ulauncher.api.server.ExtensionController: _send_event() | Send event PreferencesEvent to "com.github.isacikgoz.ukill"`
`2020-08-25 09:14:24,802 | ERROR | ulauncher.api.server.ExtensionRunner: _run_process() | Extension "com.github.fougner.ulauncher-pidgin" exited instantly with code 1`
`2020-08-25 09:14:24,802 | ERROR | ulauncher.api.server.ExtensionRunner: _run_process() | Extension "com.github.fougner.ulauncher-pidgin" failed with an error:dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name im.pidgin.purple.PurpleService was not provided by any .service files`

**Comment by fougner** [comment]
The code handling the dbus shouldn't even be executed until you run a query, so I'm not sure how it could throw anything. Are you sure you updated the plugin?

**Comment by monstrodopantano** [comment]
Yes, I updated the plugin yet the problem occurs. What I see is that before loading the Ulauncher the pidgin has to be open first. Can you solve this?.
And congratulations on the plugin is very useful.

**Comment by fougner** [comment]
I'd need a stacktrace to actually know what's going on, please try starting the extension independently as described here http://docs.ulauncher.io/en/latest/extensions/debugging.html 

That way you should get a stracktrace so I can see why it's crashing.
